tests/conftest.py

```diff
@@ -100,3 +101,68 @@ def select_by_name(dash_duo, selector, name):
 def get_options(dash_duo, selector):
     parameter_selector_input = dash_duo.find_element(selector)
     return parameter_selector_input.text.split("\n")
+
+
+def setup_plugin(dash_duo, name, pluginClass, window_size=(1024, 3600)):
```

Comment:
Seem to be very similar to `tests/data/spe1_st/tests/test_webviz_ert.py: _gen_plugin` maybe try to also replace `_gen_plugin` with this new function to keep code duplication minimal.

Reply:
thank you, i had seen it, and was later looking for it - but didn't immediately find it :sweat_smile: refactored as you suggested!